playbooks/ansible-cloud/appliance-base/pre.yaml

```diff
@@ -1,54 +1,82 @@
+# TODO DUP of ansible-test-network-integration-base/pre.yaml
```

Comment:
This is the wrong file, the file that will be dupe will be

playbooks/ansible-test-cloud-integration-base/pre.yaml 

We don't need to modify anything here.